src/transformStateWithClones.js

```diff
@@ -8,6 +8,32 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const newObj = { ...state };
+  const resArr = [];
```

Comment:
Let's use cleaner names for the variables.
Explain what resArr includes
![image](https://user-images.githubusercontent.com/96409563/220188179-55b3936a-f769-409b-8941-c9eb08df7e4d.png)